packages/odf/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/capacity-autoscaling.tsx

```diff
@@ -21,22 +23,29 @@ import {
 import './capacity-autoscaling.scss';
 import { InfoCircleIcon } from '@patternfly/react-icons';
 
-const getCapacityLimitAmount = (capacityLimit: string) =>
-  Number(capacityLimit.replace(/\D/g, ''));
-
-const getItem = (value: number) => ({
-  value: `${value}${StorageSizeUnit.Ti}`,
-  children: getFormattedCapacity(`${value}${StorageSizeUnit.Ti}`),
-});
+const getItem = (capacityLimit: number) => {
+  const limit = Number.isInteger(capacityLimit)
+    ? capacityLimit
+    : capacityLimit.toFixed(2);
+  return {
+    'data-limit': limit,
```

Comment:
shouldn't this be this as well convention ??
```suggestion
    dataLimit: limit,
```

Reply:
As per our conversation, these items later become HTML elements so I prefer to store this data in a standard `data-*` attribute.